src/main/kotlin/com/geospock/pubvote/GroupSplitter.kt

```diff
@@ -9,13 +11,25 @@ import java.util.Collections
 class GroupSplitter(private val maxGroupSize: Int) {
 
     fun splitGroups(attending: Group): List<Group> {
-        val groups = (attending.size + maxGroupSize - 1) / maxGroupSize
-
-        val listToSplit = attending.toMutableList()
-        Collections.shuffle(listToSplit)
-        return listToSplit.withIndex()
-                .groupBy {
-                    it.index % groups
-                }.map { it.value.map { it.value } }
+        val groups = (1 .. (attending.size + maxGroupSize - 1) / maxGroupSize)
+                .map { mutableListOf<People>() }
+
+        groups.addFromRole(attending, Role.ELT)
+        groups.addFromRole(attending, Role.CSMT)
+        groups.addFromRole(attending, Role.ESMT)
+        groups.addFromRole(attending, Role.PEON)
```

Comment:
alternatively
```
        with(groups) {
            addFromRole(attending, Role.ELT)
            addFromRole(attending, Role.CSMT)
            addFromRole(attending, Role.ESMT)
            addFromRole(attending, Role.PEON)
        }
```

Reply:
I don't like the use of acronyms - as they're enums it doesn't cost anything to write use `Role.EXECUTIVE_LEADERSHIP_TEAM`.